### PR TITLE
Replace vary user-agent w/ cache-control max-age=0

### PR DESCRIPTION
**Pull request by jgmize**

### CODE REVIEW

**Comment by jgmize** [comment]
@pmclanahan r?

File: bedrock/firefox/views.py [hunk 1]
```diff
@@ -284,7 +284,7 @@ class LatestFxView(TemplateView):
     """
     non_fx_redirect = 'firefox.new'
 
-    @vary_on_headers('User-Agent')
+    @cache_control(max_age=0)
```
Comment (pmclanahan) [inline bedrock/firefox/views.py]: NIT: why not use the `never_cache` decorator instead. IIRC they should have the same result, but it's a bit more explicit.

**Comment by pmclanahan** [comment]
Apart from that one nit this looks good. r+

File: bedrock/firefox/views.py (hunk 1 above)
Comment (jgmize) [inline bedrock/firefox/views.py]: I considered using the never_cache decorator, but then out of curiosity I looked at the source and saw that it calls [patch_cache_control](https://github.com/django/django/blob/1f29164ced01bde92c72d20d67bc453484a4f078/django/utils/cache.py#L41-L87) which was not necessary for this case, so I used this method as an optimization. Given the traffic levels of this particular view, I felt the optimization was justified over the very slight loss of readability.

Comment (pmclanahan) [inline bedrock/firefox/views.py]: calling `patch_cache_control` is what the `cache_control` decorator does too. They seem identical really, just that the `cache_control` one is more about setting it for some > 0 time.

**Pull request merged**